diff --git a/src/Form/AutoEntityLabelForm.php b/src/Form/AutoEntityLabelForm.php
index b9e5e4f..ba83d5c 100644
--- a/src/Form/AutoEntityLabelForm.php
+++ b/src/Form/AutoEntityLabelForm.php
@@ -134,7 +134,7 @@ class AutoEntityLabelForm extends ConfigFormBase {
     // Display the list of available placeholders if token module is installed.
     if (\Drupal::moduleHandler()->moduleExists('token')) {
       $form['auto_entitylabel']['token_help'] = array(
-        '#theme' => 'token_tree',
+        '#theme' => 'token_tree_link',
         '#token_types' => array($entity_type_parameter),
         '#dialog' => TRUE,
       );
